Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
Honestly, I think the best option is to "cheat" and use `$gitDir = cmd /c "git rev-parse --git-dir 2>nul"`, unless there are performance concerns.

If we do go the `$Error` hack route, putting that in `Invoke-Utf8ConsoleCommand` seems best.